.github/workflows/build-wheels-m1.yml

```diff
@@ -0,0 +1,45 @@
+name: Build M1 Wheels
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+    tags:
+        # NOTE: Binary build pipelines should only get triggered on release candidate builds
+        # Release candidate tags look like: v1.11.0-rc1
+        - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+  workflow_dispatch:
+
+jobs:
+  generate-matrix:
+    strategy:
+      matrix:
+        python_version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
Let's remove this part. The idea is to use only one Python version to avoid wasting resources on every PR.

Reply:
Got it thanks! So this is a test that wheels can be built, but how do we actually build them for all python versions, then?